bindings/wasm/examples/lib/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "es2022",
+    "module": "es2022",
+    "moduleResolution": "node",
```

Comment:
I'm assuming that building the `lib` directory in particular means a node user, while a vite user will be in the parent directory.